test/suite/test_verbose01.py

```diff
@@ -32,12 +32,9 @@
 import wiredtiger, wttest
 import re
 
-# test_verbose01.py
-# Tests that basic uses of the verbose configuration API work as intended i.e. passing
-# single & multiple valid and invalid verbose categories. This test is mainly focused on uses
-# of the interface prior to the introduction of verbosity levels, ensuring 'legacy'-style
-# uses of the interface are still supported.
-class test_verbose01(wttest.WiredTigerTestCase, suite_subprocess):
+# Shared base class used by verbose tests.
+class test_verbose_base(wttest.WiredTigerTestCase, suite_subprocess):
+    collection_cfg = 'key_format=S,value_format=S'
```

Comment:
I think this should live in the other class `test_verbose01` & `test_verbose02`, given its specific to the operations occurring in that test classes.

Reply:
You're right, modified in ce5fdc7f5dc687a77f551b3c675d0bc00024e874. Please note that it was already defined in `test_verbose02`, I just had to mode the definition to `test_verbose01`.